deeppavlov/core/trainers/utils.py

```diff
@@ -31,14 +31,16 @@ def parse_metrics(metrics: Iterable[Union[str, dict]], in_y: List[str], out_vars
 
         metric_name = metric['name']
         alias = metric.get('alias', metric_name)
+        kwargs = {k:metric[k] for k in set(metric)-{metric_name,alias}}
```

Comment:
You forgot about possible `inputs` key. Also kwargs is completely redundant: you can use `metric` dict.
```
metric_name = metric.pop('name')
alias = metric.pop('alias', metric_name)
```
lower inputs also can be assigned with `dict.pop` instead of `dict.get`. After that `metric` dictionary either will be empty or it will contain only parameters required by metric function.